### PR TITLE
Add explicit dependency for swss

### PR DESCRIPTION
**Pull request by qiluo-msft**

### CODE REVIEW

File: rules/swss.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 SWSS = swss_1.0.0_amd64.deb
 $(SWSS)_SRC_PATH = $(SRC_PATH)/sonic-swss
 $(SWSS)_DEPENDS += $(LIBSAIREDIS_DEV) $(LIBSAIMETADATA_DEV) $(LIBTEAM_DEV) \
-    $(LIBTEAMDCT) $(LIBTEAM_UTILS)
-$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM)
+    $(LIBTEAMDCT) $(LIBTEAM_UTILS) $(LIBHIREDIS_DEV) $(LIBSWSSCOMMON_DEV)
```
Comment (stcheng) [inline rules/swss.mk]: I think swss doesn't depend on libhiredis but swss-common depends on that.
So including swsscommon is enough.

Comment (qiluo-msft) [inline rules/swss.mk]: Fixed.

**Comment by pavel-shirshov** [comment]
Why do we need this?
How did that work before?

**Comment by stcheng** [comment]
@pavel-shirshov because swss also requires sairedis and sairedis depends on swss-common.

**Comment by qiluo-msft** [comment]
https://github.com/Azure/sonic-swss/blob/master/configure.ac#L41
swss should explicitly depend on libswsscommon.

**Comment by lguohan** [comment]
so there is about adding explicitly dependency for swss, not missing? Then, maybe you can change the commit message to clarify this.

**Pull request merged**